DataFormats/L1TMuon/src/RegionalMuonShower.cc

```diff
@@ -41,10 +43,10 @@ void l1t::RegionalMuonShower::setTFIdentifiers(int processor, tftype trackFinder
 }
 
 bool l1t::RegionalMuonShower::isValid() const {
-  return (isOneNominalInTime_ or isTwoLooseInTime_ or isOneTightInTime_);
+  return (isOneNominalInTime_ or isTwoLooseInTime_ or isOneTightInTime_ or isOneLooseInTime_);
```

Comment:
That's a good question. I would assume so, but @eyigitba should comment as that logic is performed in the EMTF. (For that matter, I think also `isOneNominalInTime_` probably implies `isOneLooseInTime_` and `isOneTightInTime_` implies all, except the two loose one.. )

Reply:
Hi @aloeliger,

Coming back to this: I thought about it some more and from my side I think I'd prefer to keep the logic "stupid", i.e. just check if any of the possible bits are set to make the shower object value, because in case something changes in the future (e.g. "tight" is in a range that doesn't include "nominal") we don't have to think to update the data format. What do you think?

Cheers,
Dinyar

Reply:
I think that's probably a good idea.

Reply:
Hi @dinyar @aloeliger , sorry for the late response here. Dinyar's comment is correct in that `isTwoLooseInTime_` should imply `isOneLooseInTime_`. Currently there is no logic in EMTF for `isOneLooseInTime_`, but it should be implemented like that. Technically the tighter cuts should always imply looser cuts are satisfied since the CSC thresholds are just simple cuts on number of hits per chamber. However, I also think that leaving it explicit like this is better for now.